azure-simple/packer/ansible-apim/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
The license file is at the root directory

Reply:
Removed